tests/test_enterprise/views/test_course_enrollment_view.py

```diff
@@ -131,14 +131,14 @@ def _check_expected_enrollment_page(self, response, expected_context):
             'discount_text': 'Discount provided by <strong>Starfleet Academy</strong>',
             'LMS_SEGMENT_KEY': settings.LMS_SEGMENT_KEY,
             'LMS_ROOT_URL': 'http://localhost:8000',
+            'course_enrollable': True,
         }
         default_context.update(expected_context)
         assert response.status_code == 200
         for key, value in default_context.items():
             assert response.context[key] == value  # pylint: disable=no-member
 
     @mock.patch('enterprise.views.render', side_effect=fake_render)
-    @mock.patch('enterprise.views.ProgramDataExtender')
```

Comment:
@UmanShahzad This is just a drive-by clean-up, right? It's not related to the main feature that this PR introduces?

Reply:
Yes, I'm noticed that we could remove this and jumped at the occasion :)